scrape_jobs.py

```
@@ -0,0 +1,78 @@
+Saves raw HTML/JSON files without LLM processing.
+"""
+
+import sys
+import argparse
+from pathlib import Path
+
+# Add app directory to Python path
+sys.path.append(str(Path(__file__).parent / "app"))
+
+from scrapers.indeed import IndeedScraper
+from scrapers.config import ScraperConfigManager, ScrapingConfig
```

Ruff (0.11.9)

`scrapers.config.ScrapingConfig` imported but unused

Remove unused import: `scrapers.config.ScrapingConfig`

(F401)